test/test-pexprs-madlibs.js

```diff
@@ -0,0 +1,195 @@
+'use strict';
+
+var test = require('tape');
+
+var makeGrammar = require('./testUtil').makeGrammar;
+
+// --------------------------------------------------------------------
+// Tests
+// --------------------------------------------------------------------
+
+// test('PExpr');
+//
+test('Alt', function(t) {
+  var g = makeGrammar([
+    'G {',
+    '  alt = "a" | "b"',
+    '}'
+  ]);
+
+  var random = Math.random;
+
+  Math.random = function() { return 0.6; };
```

Comment:
Just looking at this test, it's not clear what the significance of 0.6 and 0.2 are. Maybe add a short comment explaining why those values yield `"b"` and `"a"`, respectively?